src/java.base/share/classes/java/util/TimeZone.java

```diff
@@ -294,6 +294,10 @@ public String getID()
      * Sets the time zone ID. This does not change any other data in
      * the time zone object.
      * @param ID the new time zone ID.
+     * @implSpec The default implementation throws a
```

Comment:
Usually @implSpec follows the method description, apart from other tags. See https://openjdk.org/jeps/8068562

Reply:
Thanks, will reference this for future use of javadoc tags